requirements.txt

```diff
@@ -3,5 +3,5 @@ coverage>=4.0.3
 Sphinx>=1.3.5
 pexpect>=4.0.0
 pyyaml
-setuptools
+packaging
```

Comment:
Note: I don't know what is this file used for. Perhaps it needs setuptools listed, as a build dependency, perhaps not.